lib/rspec/core/configuration.rb

```diff
@@ -2135,6 +2135,8 @@ def load_file_handling_errors(method, file)
         suggestions = DidYouMean.new(relative_file).call
         reporter.notify_non_example_exception(ex, "An error occurred while loading #{relative_file}.#{suggestions}")
         RSpec.world.wants_to_quit = true
+      rescue SyntaxError => e
+        raise e
```

Comment:
We probably want to do the same as `SystemExit`, which outputs a normal failure message explaining why we are quitting, and then tells the rest of RSpec we are quitting.

Reply:
Thanks I’ll look to that case for guidance

Reply:
Updated. CI won't re-run until approval